test/services/pseudonym/index.test.js

```diff
@@ -1,11 +1,12 @@
-const assert = require('assert');
-const chai = require('chai');
+const { expect } = require('chai');
+const { ObjectId } = require('mongoose').Types;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4681704)